### ISSUE TITLE
run_pfam_scan.sh uses obsolete -pfamB option

### ISSUE THREAD

**Issue by peterjc** [krasileva-group/plant_rgenes#8]
According to discussion in #6 these scripts should work with recent Pfam releases using HMMER3 and a recent pfam_scan.pl.

However, the current version of ``bash_scripts/run_pfam_scan.sh`` when calling ``pfam_scan.pl`` v1.6 fails, it aborts with the message:

```
FATAL: As of release 28.0, Pfam no longer produces Pfam-B. The -pfamB and -only_pfamB options are now obsolete.
```

It is trivial to remove ``-pfamB`` from ``bash_scripts/run_pfam_scan.sh`` but does this break any downstream assumptions in the pipeline?

**Comment by krasileva** [comment]
Hi Peter,

You can safely remove it.
We did not use PfamB in downstream analyses, so it is a leftover from testing it during pfamscan step.
The downstream scripts have ‘skip PfamB’ in them, which would become unnecessary but would not break anything.

Best wishes,

Ksenia

----------------------------------------------

Ksenia Krasileva
Group Leader

[cid:image001.png@01D361F8.420F9EE0]

Norwich Research Park
Norwich
Norfolk
NR4 7UZ
+44 (0) 1603 450 2946
Ksenia.Krasileva@earlham.ac.uk<mailto:Ksenia.Krasileva@earlham.ac.uk>

www.earlham.ac.uk/krasileva-group<http://www.earlham.ac.uk/krasileva-group>
www.tsl.ac.uk/groups/krasileva-group<http://www.tsl.ac.uk/groups/krasileva-group>
@kseniakrasileva

www.earlham.ac.uk<http://www.earlham.ac.uk/>
www.tsl.ac.uk<http://www.tsl.ac.uk/>

[cid:image002.png@01D361F8.420F9EE0]<https://twitter.com/EarlhamInst>[cid:image003.png@01D361F8.420F9EE0]<https://www.facebook.com/EarlhamInst/>

**Comment by peterjc** [comment]
Thanks - that's good news.

Are you able to update this repository yourself and/or merge pull requests?

**Comment by krasileva** [comment]
Hi Peter,

Technically, yes, but I find myself without a minute to do this at the moment because I am at home with sick kids.
Would you like to join as collaborator? you have so many useful suggestions so I think the project will benefit from your formal input.

Best wishes,

Ksenia

----------------------------------------------

Ksenia Krasileva
Group Leader

[cid:image001.png@01D361FD.28901DF0]

Norwich Research Park
Norwich
Norfolk
NR4 7UZ
+44 (0) 1603 450 2946
Ksenia.Krasileva@earlham.ac.uk<mailto:Ksenia.Krasileva@earlham.ac.uk>

www.earlham.ac.uk/krasileva-group<http://www.earlham.ac.uk/krasileva-group>
www.tsl.ac.uk/groups/krasileva-group<http://www.tsl.ac.uk/groups/krasileva-group>
@kseniakrasileva

www.earlham.ac.uk<http://www.earlham.ac.uk/>
www.tsl.ac.uk<http://www.tsl.ac.uk/>

[cid:image002.png@01D361FD.28901DF0]<https://twitter.com/EarlhamInst>[cid:image003.png@01D361FD.28901DF0]<https://www.facebook.com/EarlhamInst/>

**Comment by peterjc** [comment]
I'll decline the kind GitHub collaborator status offer (for now at least), but will submit a few pull requests for you to look at later when you have time back in the office - no rush.

**Comment by krasileva** [comment]
Hi Peter,

Ok. I appreciate all of your comments!

Best wishes,

Ksenia

----------------------------------------------

Ksenia Krasileva
Group Leader

[cid:image001.png@01D36204.161BFAC0]

Norwich Research Park
Norwich
Norfolk
NR4 7UZ
+44 (0) 1603 450 2946
Ksenia.Krasileva@earlham.ac.uk<mailto:Ksenia.Krasileva@earlham.ac.uk>

www.earlham.ac.uk/krasileva-group<http://www.earlham.ac.uk/krasileva-group>
www.tsl.ac.uk/groups/krasileva-group<http://www.tsl.ac.uk/groups/krasileva-group>
@kseniakrasileva

www.earlham.ac.uk<http://www.earlham.ac.uk/>
www.tsl.ac.uk<http://www.tsl.ac.uk/>

[cid:image002.png@01D36204.161BFAC0]<https://twitter.com/EarlhamInst>[cid:image003.png@01D36204.161BFAC0]<https://www.facebook.com/EarlhamInst/>

**Comment by peterjc** [comment]
Fixed via #9
